resync/api/middleware/cors_config.py

```
@@ -196,59 +124,36 @@ def _is_valid_origin_format(origin: str) -> bool:
             True if format is valid, False otherwise
         """
-        if origin == "*":
+        if origin == '*':
             return True
-
-        # Parse the origin using urlparse for proper validation
         try:
             parsed = urlparse(origin)
         except Exception as e:
-            logger.error("exception_caught", error=str(e), exc_info=True)
+            logger.error('exception_caught', exc_info=True, extra={'error': str(e)})
```

Ruff (0.15.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)